release.properties

```diff
@@ -53,11 +53,11 @@
 
     #release.info=on
 
-    release.info.build=7201
+    release.info.build=7300
     release.info.date=November 12, 2019
     release.info.name=Liferay Community Edition Portal
-    release.info.version=7.2.1
-    release.info.version.display.name=7.2.1 CE GA2
+    release.info.version=7.3.0
+    release.info.version.display.name=7.3.0 CE M1
```

Comment:
Please change to `7.3.0 CE GA1`. We won't have pre-releases for 7.3 (a lot of GAs instead)